docker-compose.yml

```diff
@@ -26,21 +26,26 @@ services:
             - DJANGO_SECURE_SESSION=False
             - FHIR_URL="https://prod-sbx.bfd.cms.gov"
             - DJANGO_FHIR_CERTSTORE=/code/docker-compose/certstore/
-            # - DJANGO_MEDICARE_SLSX_LOGIN_URI=http://127.0.0.1:8080/sso/authorize?client_id=bb2api
+            # Commented out settings for MSLS:
             # - DJANGO_MEDICARE_SLSX_REDIRECT_URI=http://localhost:8000/mymedicare/sls-callback
-            # - DJANGO_SLSX_USERINFO_ENDPOINT=http://msls:8080/v1/users
+            # - DJANGO_MEDICARE_SLSX_LOGIN_URI=http://127.0.0.1:8080/sso/authorize?client_id=bb2api
             # - DJANGO_SLSX_HEALTH_CHECK_ENDPOINT=http://msls:8080/health
             # - DJANGO_SLSX_TOKEN_ENDPOINT=http://msls:8080/sso/session
             # - DJANGO_SLSX_SIGNOUT_ENDPOINT=http://msls:8080/sso/signout
+            # - DJANGO_SLSX_USERINFO_ENDPOINT=http://msls:8080/v1/users
+            # - DJANGO_SLSX_VERIFY_SSL_INTERNAL=False
```

Comment:
may be these 2 lines could be removed since it does not affect mslsx endpoints - they are all http, tested locally without the 2 env vars for mslsx, and it works as is.

Reply:
> may be these 2 lines could be removed since it does not affect mslsx endpoints - they are all http, tested locally without the 2 env vars for mslsx, and it works as is.

@JFU-GIT Good catch. These are not needed for msls. I just removed them.